subsys/net/lib/lwm2m/lwm2m_obj_connmon.c

```diff
@@ -95,7 +103,11 @@ static struct lwm2m_engine_obj_field fields[] = {
 	OBJ_FIELD_DATA(CONNMON_APN, R_OPT, STRING),
 	OBJ_FIELD_DATA(CONNMON_CELLID, R_OPT, U32),
 	OBJ_FIELD_DATA(CONNMON_SMNC, R_OPT, U16),
-	OBJ_FIELD_DATA(CONNMON_SMCC, R_OPT, U16)
+	OBJ_FIELD_DATA(CONNMON_SMCC, R_OPT, U16),
```

Comment:
Your changes look OK to me. Additionally, I have a similar comment as with the PR #41433 - all the resources with numerical values should be signed integers instead of unsigned ones.

![image](https://user-images.githubusercontent.com/33285290/147541671-457fb278-6a35-4939-be16-192bc475310e.png)

![image](https://user-images.githubusercontent.com/33285290/147541798-cbf87a17-68a4-487e-9d27-b5237c04cd1c.png)